src/SignalR/server/SignalR/test/Internal/MessageBufferTests.cs

```diff
@@ -235,6 +235,43 @@ public async Task WriteManyMessagesAckSomeProperlyBuffers()
             pipes.Application.Input.AdvanceTo(buffer.Start);
         }
     }
+
+    [Fact]
+    public async Task MessageBufferLimitCanBeModified()
```

Comment:
I might be missing it but I'm not seeing where the message buffer limit is modified here.

Reply:
Oh wait ... you are changing it from the default :) Duh!.